sdk/sdk-skeleton.go

```diff
@@ -0,0 +1,126 @@
+import "time"
+
+type Asset struct{}
+
+//// Methods directly on the SDK
+
+func NewChannel() {
+
+}
+
+func NewChannelListenerService() {
+
+}
+
+func GetChannel() {
+
+}
```

Comment:
I think `GetChannel` was only required when the factory existed as well. I think you can remove it.

Reply:
https://github.com/stellar/experimental-payment-channels/pull/25/commits/a7322ba7b51ff9ecf88e807adc9401641821a8a3